configuration/configuration.go

```diff
@@ -86,6 +86,12 @@ type Configuration struct {
 			ClientCAs []string `yaml:"clientcas,omitempty"`
 		} `yaml:"tls,omitempty"`
 
+		// Headers is a set of headers to include in HTTP responses. A common
+		// use case for this would be security headers such as
+		// Strict-Transport-Security. The map keys are the header names, and
+		// the values are the associated header payloads.
+		Headers map[string]string `yaml:"headers,omitempty"`
```

Comment:
No, actually, `Endpoint` has a `Headers   http.Header` field. It awkward but complete:

```yaml
headers:
    My-Header: [value1] # commonly used for a single header value.
    My-Header2: 
        - foo
        - bar
```